pkgs/development/libraries/glog/default.nix

```diff
@@ -20,9 +20,18 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  postPatch = lib.optionalString stdenv.isDarwin ''
+    # A path clash on case-insensitive file systems blocks creation of the build directory.
+    # The file in question is specific to bazel and does not influence the build result.
+    rm BUILD
+  '';
+
   nativeBuildInputs = [ cmake ];
 
-  cmakeFlags = [ "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}" ];
+  propagatedBuildInputs = [ gflags ];
+
+  cmakeFlags = [ "-DBUILD_SHARED_LIBS=${if static then "OFF" else "ON"}" ]
+    ++ lib.optional stdenv.isDarwin "-DCMAKE_POLICY_DEFAULT_CMP0068=NEW";
```

Comment:
Was this set this for any specific reason?

Reply:
CMake prints a warning about [this policy](https://cmake.org/cmake/help/latest/policy/CMP0068.html). Now that I think of it the old behavior is probably preferred in this case.

Reply:
Well, from what I read [1], the new behaviour expects us to make a decision about the "BUILD_WITH_INSTALL_NAME_DIR". Let's not touch this in this PR.

[1] https://gitlab.kitware.com/cmake/cmake/commit/68c6ffde9f61814cce14779e95bf0ef90a2b5a12

Reply:
Ok, it should probably be done in the `setup-hook` anyway.